ts/tests/testParser.ts

```diff
@@ -0,0 +1,100 @@
+import parser from "../parser";
+import { expect, assert } from "chai";
```

Comment:
Am I weird for being concerned that `chai` provides _too much_ syntactic sugar? How am I supposed to decide between

```js
assert.isFalse(parser.isLabel(line));
assert.isNotTrue(parser.isLabel(line));
assert.equal(parser.isLabel(line), false);
assert.strictEqual(parser.isLabel(line), false);
assert.isNotOk(parser.isLabel(line));
expect(parser.isLabel(line)).to.be.false;
expect(parser.isLabel(line)).to.equal(false);
expect(parser.isLabel(line)).to.not.be.ok;
```

?

We should at least pick one of `expect` and `assert`, which seem to provide equivalent sets of functionality.

We could also consider whether Node’s [built-in `assert` module](https://nodejs.org/api/assert.html) (in strict mode: `import { strict as assert } from "assert"`) is good enough.

Reply:
That syntax is quite silly. I prefer chai assert because it supports assert.isFalse a bit better than Node's built-in assert module. I'll just go with assert everywhere. 🤷‍♀️